AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -103,6 +103,7 @@ postoffice.co.uk###ensBannerBG
 bdb.com.pl###plansza
 superuser.com##.js-consent-banner
 rasthaber.com##.rasthaber-cookie
+affinity.serif.com###elGuestTerms
```

Comment:
Is is unblocked on your device?

Reply:
I was able to reproduce it on firefox mobile